Intro to Python Notebook #1 .ipynb

```diff
@@ -0,0 +1,1718 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Go Duke Devils go!\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('Go Duke Devils go!')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial is in a jupyter notebook. However, python can also be written in text files. If you run the command \"python\" on a file whose name ends in *.py* then Python will execute all of the code in the file. You can try this out by opening up a terminal, cd'ing to this directory and typing \"python hello_world.py\" "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part 2: Variables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Typing phrases is fun, but our ultimate goal is to be able to use python for science and computation. Just as we did in computation by hand, we can start by defining variables. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Variables can be assigned and name and a value with the \"=\" sign: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "h_bar = 1.05457e-34"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1.05457e-34\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(h_bar)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once variables are assigned a value, they can be manipulated however we want: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "6.6260531326e-34\n"
+     ]
+    }
+   ],
+   "source": [
+    "pi = 3.14159\n",
+    "h = 2*pi*h_bar\n",
+    "print(h)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "All variables are a certain type that dictates how it is used. Each type has different properties."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Here are some examples\n",
+    "# By the way, the # symbol is a way to add a comment, these lines will not be run by the \n",
+    "# interpreter, and serve to add context to the code being run \n",
+    "\n",
+    "x=1 #int, only digits\n",
+    "x=1.0 #float, because of the '.', note that re-defining x in this way resets it to a new value\n",
+    "x=1.05457e-34 # also a float because of the '.' and 'e'\n",
+    "x='Duke Devils' #str, defined by quotes around text\n",
+    "x=True #bool"
```

Comment:
Just a matter of style, probably better to use a space before and after `=`. Also having two spaces before inline comment and a space after `#`.